docs/ref/csrf.txt

```diff
@@ -462,3 +472,30 @@ A number of settings can be used to control Django's CSRF behavior:
 * :setting:`CSRF_FAILURE_VIEW`
 * :setting:`CSRF_HEADER_NAME`
 * :setting:`CSRF_TRUSTED_ORIGINS`
+
+Frequently Asked Questions
+==========================
+
+Is posting an arbitrary CSRF token pair (cookie and POST data) using Firebug, Chrome dev tools, Burp Suite, etc. a vulnerability?
+---------------------------------------------------------------------------------------------------------------------------------
+
+No, this is by design. The comparison between the cookie token and the POST
+token is a server-side check but it's not against any stored server-side value.
+Some security audits flag this as a problem but remember that an attacker
+cannot steal a user's browser's CSRF cookie. "Stealing" or modifying your
+own token isn't a vulnerability.
+
+Is the fact that Django's CSRF protection isn't linked to a session a problem?
+------------------------------------------------------------------------------
+
+No, this is by design. We may link them in the future just so that we don't
```

Comment:
I know where this is coming from, but I think the answer should go into the details of "design". First of all (imagine a pastebin) you'd want to prevent any other site posting to you via the victims browser without them knowing -- so it is just not feasible to require a session. Maybe @PaulMcMillan / @dstufft can provide examples in which cases linking to the session would actually provide a real benefit (we are rotating the token on login anyways and assuming https, I do not see any real gain).